packages/perseus-editor/src/components/interactive-graph-settings.tsx

```diff
@@ -32,6 +34,35 @@ function numSteps(range: any, step: any) {
     return Math.floor((range[1] - range[0]) / step);
 }
 
+function Heading({
+    title,
+    isOpen,
+    onToggle,
+}: {
+    title: string;
+    isOpen: boolean;
+    onToggle?: (isOpen: boolean) => void;
+}) {
+    return (
+        <View
+            style={{
+                flexDirection: "row",
+                justifyContent: "space-between",
+                backgroundColor: color.offBlack8,
+                padding: spacing.xSmall_8,
+                marginTop: spacing.small_12,
+                marginLeft: -10,
+                marginRight: -10,
```

Comment:
Switched to `marginInline`, thanks. 

For now, this must be exactly -10 because it negates the `@editorPadding` Less variable from `perseus-editor.less`.

Reply:
I added a comment. But I'll reproduce it here. The value is the inverse of a Less variable `@editorPadding`. If we moved the value to be a CSS variable, we could use it in both the Less file `var(--editor-padding)` as well as in this Aphrodite style (`marginInline: calc(-1 * --editor-padding)`. 

I haven't done that in this PR though as I think we'd want to talk about if that's a way we want to move forward before I start a new pattern.